website/templates/project/settings.mako

```diff
@@ -387,6 +411,8 @@
       window.contextVars.node.description = ${node['description'] | sjson, n };
       window.contextVars.node.nodeType = ${ node['node_type'] | sjson, n };
       window.contextVars.nodeCategories = ${ categories | sjson, n };
+      window.contextVars.wiki = window.contextVars.wiki || {};
```

Comment:
There.  I just need the enabled or disabled information.  I created the object to mimic the pseudocode you created and also thinking the object could come in handy later if and when I want to add additional attributes when you can change all the wiki permissions of the children.